flink-clients/src/test/java/org/apache/flink/client/cli/CliFrontendModifyTest.java

```diff
@@ -115,13 +115,12 @@ public void testUnparsableJobId() throws Exception {
 		return rescaleJobFuture.get();
 	}
 
-	private static final class TestingClusterClient extends StandaloneClusterClient {
+	private static final class TestingClusterClient extends RestClusterClient<StandaloneClusterId> {
```

Comment:
Does this remove test coverage for the old client?

Reply:
[Stephan Ewen on ML](https://lists.apache.org/thread.html/cc46fde6f8b31d4e833b01e5814a9547c8a67ea3e08a31ec5d71145e@%3Cdev.flink.apache.org%3E) mentioned that we will finally use a HTTP client. Thus we can only check `RestClusteClient`.

Also [FLINK-11540](https://issues.apache.org/jira/browse/FLINK-11540) intends to remove `StandaloneClusterClient` since it is legacy now.

Reply:
Thanks, makes sense.